serializer/src/test/java/net/md_5/bungee/api/chat/ComponentsTest.java

```diff
@@ -868,4 +868,10 @@ private static String fromAndToLegacyText(String legacyText)
     {
         return BaseComponent.toLegacyText( TextComponent.fromLegacyText( legacyText ) );
     }
+
+    @Test
+    public void testArrayParsing()
+    {
+        System.out.println( ComponentSerializer.deserialize( "[Outfluencer,[\" \",is,[\" very\",\" cool \",[b,dfg28dhz,[c,[a,thisisacoolcomponent]]]]]]" ).toPlainText() );
```

Comment:
Can we make this an assertion

Reply:
Ah yes i didn't even wanted to print that, it was just for testing. You can change it, or i will do it tomorrow